main.rb

```diff
@@ -51,13 +28,12 @@ def h(text)
 end
 
 post '/memos' do
-  memo = { 'id' => build_id, 'title' => params['title'], 'content' => params['content'], 'created_at' => Time.now }
-  save_memo(memo['id'], memo)
-  redirect "/memos"
+  Memo.create(params)
```

Comment:
JSON時代のように、
```suggestion
  memo = {...}
  Memo.create(memo)
```

みたいな形にするとparamsへの再代入も気にしなくて済むかもしれません。